core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -455,6 +458,13 @@ public long maxConnectionAgeMillis() {
         return maxConnectionAgeMillis;
     }
 
+    /**
+     * Returns the connection shutdown grace period.
+     */
+    public Duration connectionShutdownGracePeriod() {
```

Comment:
Could be replaced with `public long connectionShutdownGracePeriodMillis()` so it's consistent with other millis-returning methods. We could return nanos if nanoseconds precision is necessary, but I guess that's probably not the case this time?

Reply:
I don't think nanos are needed, but microseconds might be, see https://github.com/line/armeria/pull/3715/files#r676900302.